### ISSUE TITLE
Empty results

### ISSUE THREAD

**Issue by kovinet** [JsusDev/yandex-pictures#1]
I have tried running this and first time it worked, but when I ran it couple of times I am just getting empty array as a result.

Could it be because Yandex is blocking the crawler?

**Comment by kovinet** [comment]
If I change my IP it again works for couple of times but then stops, which is sign that Yandex is blocking this crawler.

**Comment by JsusDev** [comment]
@kovinet Yes, Yandex requires proxy request, otherwise there is a lock, I did not have time to add a proxy at the time of release

**Comment by c00ler123** [comment]
You can use proxy-lists package
